ui/src/cloud/utils/overrides.ts

```diff
@@ -0,0 +1,33 @@
+import _ from 'lodash'
+import HoneyBadger from 'honeybadger-js'
+import {GIT_SHA} from 'src/shared/constants'
+
+if (process.env.CLOUD === 'true') {
+
+  HoneyBadger.configure({
+    apiKey: process.env.HONEYBADGER_KEY,
+    revision: GIT_SHA,
+    environment: process.env.HONEYBADGER_ENV,
```

Comment:
This accepts a context where we could add more helpful information like user_id. Or in the honeybadger client we could set context using the honeybadger.resetContext or .addContext